### ISSUE TITLE
Arguments should not be marked required. Conflicts with 'justInclude' parameter.

### ISSUE THREAD

**Issue by dammejed** [development-bizeta-mestre/nsis-extension#4]
Hello,

When attempting to use this task in 'justInclude' mode, I get an error when I don't provide the 'arguments' parameter. This is superfluous for justInclude, because makensis won't even be invoked by the task.

Here's the input:
```yaml
- task: bizeta.nsis-task.796c90b1-9f8c-424a-b03b-b789bb5945d3.nsis@1
  displayName: 'Include Nsis'
  inputs:
    justInclude: yes
    includeMorePlugins: yes
```

And the resulting task output:
```
Starting: Include Nsis
==============================================================================
Task         : Nsis
Description  : Run Nsis
Version      : 1.1.7
Author       : bizeta
Help         : Run Nsis [More Information](https://github.com/dev-maxima/nsis-extension)
==============================================================================
##[error]Required: 'arguments' input
Finishing: Include Nsis
```

**Comment by dzoccarato** [comment]
Hi @dammejed, sorry for this delay... we don't check often this repo.
Now $arguments is no longer mandatory as parameter for this task.
But still required when $justInclude is set to 'no'.

PR: #9, extension and published to 1.1.8

**Comment by dzoccarato** [comment]
Hi @dammejed, should resolved with last release 1.1.9.

Thanks!
